src/server/cea_608_encoder/caption_string_utility.py

```diff
@@ -141,20 +144,44 @@ def create_byte_pairs_for_caption_string(caption_string: str) -> list:
     byte_pairs = bytes_to_byte_pairs(raw_hex_values)
     return byte_pairs
 
+def create_byte_pairs_for_midrow_style(color: str, underline = False):
+    byte_list = []
+    
+    # Default: text color: white, no underline
+    first_byte = 0x11 
+    second_byte = 0x20
+
+    if isinstance(color, str):
+        color = color.lower()
+    if color in text_colors:
+        second_byte += text_colors[color]
+    if underline == True: 
+        second_byte += 0x1  
+
+    bytes = []
+    bytes.append(first_byte)
+    bytes.append(second_byte)
+
+    # To move the cursor one position back
+    # CC1 Backspace control command
+    bytes.append(0x14)
```

Comment:
Can you create a function for obtaining these values. something like get_single_backspace_bytes or something more clever

Reply:
Done